samples/getAutoclass.js

```diff
@@ -37,7 +37,7 @@ function main(bucketName = 'my-bucket') {
   async function getAutoclass() {
     const [metadata] = await storage.bucket(bucketName).getMetadata();
     console.log(
-      `Autoclass enabled is set to ${metadata.autoclass.enabled} for ${metadata.name} at ${metadata.autoclass.toggleTime}.`
+      `Autoclass terminal storage class is set to ${metadata.autoclass.terminalStorageClass} for ${metadata.name} at ${metadata.autoclass.terminalStorageClassUpdateTime}.`
```

Comment:
For the getAutoclass sample, I think it'd be nice to demonstrate all four fields in the Autoclass config. Something along the lines of
```
console.log( 
  `Autoclass is ${metadata.autoclass.enabled ? 'enabled' : 'disabled'} for ${metadata.name} at ${metadata.autoclass.toggleTime}.
   Autoclass terminal storage class is last updated to ${metadata.autoclass.terminalStorageClass} at ${metadata.autoclass.terminalStorageClassUpdateTime}.`
)
```

Reply:
Done